service/src/main/kotlin/fi/espoo/evaka/vasu/OphQuestion.kt

```diff
@@ -849,17 +852,6 @@ fun getDefaultLeopsContent(lang: VasuLanguage) = VasuContent(
                 )
             )
         ),
-        VasuSection(
```

Comment:
As the vasu/LEOPS e2e tests are unfortunately dependent on the rendered question numbers, the removal of this section should be reflected in the LEOPS e2e test as well (namely decreasing the numbers by 1).